lib/commons/dom/url-props-from-attribute.js

```diff
@@ -8,7 +8,9 @@
  * @returns {Object}
  */
 dom.urlPropsFromAttribute = function urlPropsFromAttribute(node, attribute) {
-	const value = node[attribute];
+	const value = !node.ownerSVGElement
+		? node[attribute]
+		: node.getAttribute(attribute);
```

Comment:
Didn't fix the problem. `href=""` for SVG will still result in `undefined` getting returned by this function.